bazel/BUILD

```diff
@@ -248,6 +248,16 @@ config_setting(
     values = {"define": "quiche=enabled"},
 )
 
+config_setting(
```

Comment:
we decided not upstream WAVM meanwhile?

Reply:
The performance of WAVM is much higher than V8, consequently the istio team would like to have WAVM as an option and we felt that if istio would want it others would as well.

Reply:
We discussed this with Matt at our last meeting  (you might have come in after that) and he was onboard with including WAVM as an option.

Reply:
Sounds good then. Do we need this config_settings though? Isn't VM implementations are registered via extensions?

Reply:
That is a good point.  Currently it is implemented as a -define wasm=XXXX option.  That was appropriate when we were doing development, but perhaps registering it as an extension would be more appropriate now. I am not sure how that would work though.  Perhaps using a envoy/registry/registry.h and registering a factory for the runtime?

Reply:
yes, feel free to just leave a TODO (with an issue) and tackle it later.

Reply:
Done.  With issue https://github.com/envoyproxy/envoy/issues/12574